example/lib/line_chart/samples/line_chart_sample5.dart

```diff
@@ -144,12 +144,17 @@ class LineChartSample5 extends StatelessWidget {
 
 /// Lerps between a [LinearGradient] colors, based on [t]
 Color lerpGradient(List<Color> colors, List<double> stops, double t) {
+  if (colors.isEmpty) {
+    return Colors.green;
```

Comment:
I think it is better to throw an Exception instead of returning a hard-coded value.

Reply:
done